website/src/pages/v3/comparison.mdx

```diff
@@ -1,7 +1,6 @@
 ---
```

Comment:
I guess this is not needed at all. I see `#` is used in v2 docs instead of `title`

Reply:
I will take care of the deviations from [v2](https://github.com/dotansimha/graphql-yoga/tree/v2) in a separate PR.